ckan/lib/base.py

```diff
@@ -105,7 +87,7 @@ def render_jinja2(template_name, extra_vars):
 
 
 def render(template_name, extra_vars=None, cache_key=None, cache_type=None,
-           cache_expire=None, method='xhtml', loader_class=MarkupTemplate,
+           cache_expire=None, method='xhtml',
```

Comment:
We can probably get rid of `method` as well, as the Jinja `render` function won't use it